package.json

```diff
@@ -527,21 +526,22 @@
   "scripts": {
     "vscode:prepublish": "pwsh -file ./build.ps1 build",
     "compile": "tsc -p ./",
-    "watch": "pwsh -file ./build.ps1 initial && tsc -watch -p ./",
+    "watch": "tsc -watch -p ./",
     "pretest": "npm run compile",
     "test": "node ./out/test/runtest.js"
   },
   "devDependencies": {
     "@types/glob": "^7.1.1",
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.0.8",
-    "@types/vscode": "^1.30.0",
+    "@types/vscode": "^1.42.0",
```

Comment:
Note to self, need to find lowest vscode version that supports webview uri

Reply:
The lowest version that supports the `webview.asWebviewUri` is `1.38`. This means raising the minimum engine version to `1.38`.